### PR TITLE
update the QUICKSTART for 0.15.0

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -122,15 +122,15 @@ void b(Span parentSpan) {
 ```
 The OpenTelemetry API offers also an automated way to propagate the parent span on the current thread:
 ```java
-void a() {
+void c() {
   Span parentSpan = tracer.spanBuilder("a").startSpan();
```
Comment (anuraaga) [inline QUICKSTART.md:126]:
```suggestion
  Span parentSpan = tracer.spanBuilder("c").startSpan();
```

File: QUICKSTART.md [hunk 2]
```diff
@@ -122,15 +122,15 @@ void b(Span parentSpan) {
...
   try(Scope scope = parentSpan.makeCurrent()) {
     b();
```
Comment (anuraaga) [inline QUICKSTART.md:128]:
```suggestion
    d();
```

File: QUICKSTART.md [hunk 3]
```diff
@@ -122,15 +122,15 @@ void b(Span parentSpan) {
...
   } finally {
     parentSpan.end();
   }
 }
-void b() {
+void d() {
   Span childSpan = tracer.spanBuilder("b")
```
Comment (anuraaga) [inline QUICKSTART.md:134]:
```suggestion
  Span childSpan = tracer.spanBuilder("d")
```

File: QUICKSTART.md [hunk 4]
```diff
@@ -381,13 +375,12 @@ Additional samplers can be provided by implementing the `io.opentelemetry.sdk.tr
...
 
 ```java
-TraceConfig alwaysOn = TraceConfig.getDefault().toBuilder().setSampler(Sampler.alwaysOn()).build();
-TraceConfig alwaysOff = TraceConfig.getDefault().toBuilder().setSampler(Sampler.alwaysOff()).build();
-TraceConfig half = TraceConfig.getDefault().toBuilder().setSampler(Sampler.traceIdRatioBased(0.5)).build();
-
-// You configure the sampler when building the SDK implementation:
     SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
-      .setTraceConfig(half)
+      .setSampler(Sampler.alwaysOn())
```
Comment (anuraaga) [inline QUICKSTART.md:379]: Nice

File: QUICKSTART.md [hunk 5]
```diff
@@ -401,11 +394,11 @@ in bulk. Multiple Span processors can be configured to be active at the same tim
 ```java
     SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
       .addSpanProcessor(
-        MultiSpanProcessor.create(Arrays.asList(
+        SpanProcessor.composite(
```
Comment (anuraaga) [inline QUICKSTART.md:397]: I guess multiple calls to `addSpanProcessor` would be more natural, and ends up with us creating the composite

File: QUICKSTART.md [hunk 6]
```diff
@@ -421,14 +414,17 @@ a particular backend. OpenTelemetry offers five exporters out of the box:
...
-ManagedChannel jaegerChannel =
-    ManagedChannelBuilder.forAddress([ip:String], [port:int]).usePlaintext().build();
+    ManagedChannel jaegerChannel = ManagedChannelBuilder.forAddress("localhost", 3336)
+      .usePlaintext()
+      .build();
+    
     JaegerGrpcSpanExporter jaegerExporter = JaegerGrpcSpanExporter.builder()
-      .setServiceName("example").setChannel(jaegerChannel).setDeadline(30000)
-      .build()
+      .setChannel(jaegerChannel).setTimeout(30, TimeUnit.SECONDS)
```
Comment (anuraaga) [inline QUICKSTART.md:422]: How about updating this to `setEndpoint("localhost:3336")`?

**Review by anuraaga — APPROVED** [review]

File: QUICKSTART.md [hunk 7]
```diff
@@ -122,16 +122,16 @@ void b(Span parentSpan) {
...
-    b();
+    d();
   } finally {
     parentSpan.end();
   }
 }
-void b() {
-  Span childSpan = tracer.spanBuilder("b")
+void d() {
+  Span childSpan = tracer.spanBuilder("d")
```
Comment (bogdandrutu) [inline QUICKSTART.md:134]: Please avoid 1 character function names in examples, also using `b` and `d` makes things even harder to see.

Comment (jkwatson) [reply QUICKSTART.md:134]: I thought you liked c!  ;) 

I'll clean this up.

Comment (jkwatson) [reply QUICKSTART.md:134]: By the way, the only reason this even got touched is that I'm starting to think about auto-generating these docs from a template java file, and I needed it all to compile locally. :)

File: QUICKSTART.md (hunk 5 above)
Comment (jkwatson) [reply QUICKSTART.md]: done

File: QUICKSTART.md (hunk 6 above)
Comment (jkwatson) [reply QUICKSTART.md]: done

File: QUICKSTART.md (hunk 7 above)
Comment (jkwatson) [reply QUICKSTART.md]: done

**Pull request merged**